wolfssl/wolfcrypt/types.h

```diff
@@ -451,9 +452,11 @@
 	};
 
 	/* max error buffer string size */
+    #ifndef WOLFSSL_MAX_ERROR_SZ
```

Comment:
I recommend changing this to a define instead of enum. That was it’s always consistent.